yt/frontends/openPMD/data_structures.py

```diff
@@ -193,7 +206,7 @@ def _populate_grid_objects(self):
         self.max_level = 0
 
 
-class openPMDDataset(Dataset):
+class openPMDDataset(Dataset, openPMDBasePath):
     """
     A dataset object contains all the information of the simulation and
```

Comment:
afaik, `yt.load()` can load multiple data sets and also supports of multiple iteration-loading if done that way.